### ISSUE TITLE
Some questions about the code

### ISSUE THREAD

**Issue by Superman-Valencia** [PecholaL/MAIN-VC#3]
Thank you so much for open source code! 
I have run  the code you provided, but I have met the following questions:
(1) During training, the reconstruction loss first decreased and then increased. I am not sure whether there is a problem in the code?

(2) After I removed "-" in loss_mi = -self.mi_club.mi_est(emb, mu.transpose(1, 2)), the model converged normally, but the final performance was not as good as the demo. I wonder if my modifications are right. Or can you provide the specific number of training iterations and the final situation of convergence?

(3) Can you provide pre-trained weights for inference?

Thanks again.

**Comment by PecholaL** [comment]
(1) In my impression, compared to CMI, the reconstruction loss is easy to converge. Attached are graphs of the loss variation during 200k training process. The samples in the demo are obtained from a 200k-iter-trained model. For your situation, I suggest adjusting the weight hyperparameters.

(2) The CMI module is modified from CLUB and MINE, the code is available at https://github.com/Linear95/CLUB.

(3) There may be slight differences between the current code and the final version we executed, as the final version and ckpt are on the company's internal server and cannot be retrieved due to limitations. I apologize for this.
[CMI_train.pdf](https://github.com/PecholaL/MAIN-VC/files/15383928/CMI_train.pdf)
[klLoss.pdf](https://github.com/PecholaL/MAIN-VC/files/15383929/klLoss.pdf)
[reconLoss.pdf](https://github.com/PecholaL/MAIN-VC/files/15383930/reconLoss.pdf)
[siaLoss.pdf](https://github.com/PecholaL/MAIN-VC/files/15383931/siaLoss.pdf)
